src/current/v24.3/create-function.md

```diff
@@ -294,77 +294,84 @@ The preceding example modifies a given `name` value and returns the `NEW` [trigg
 
 The following example defines a function using the `SECURITY DEFINER` clause. This causes the function to execute with the privileges of the function owner.
 
-Create two roles:
+1. Create two roles:
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-CREATE ROLE owner;
-CREATE ROLE invoker;
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE ROLE owner;
+    CREATE ROLE invoker;
+    ~~~
 
-Grant a [`SELECT` privilege]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on the `user_promo_codes` table to the `owner` role.
+1. Grant a [`SELECT` privilege]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on the `user_promo_codes` table to the `owner` role.
```

Comment:
I'm assuming the repeated `1.` ends up being something else in the actual docs?

Reply:
Yep, this becomes a numbered list.